zerver/management/commands/realm_linkifiers.py

```diff
@@ -2,24 +2,24 @@
 from argparse import ArgumentParser
 from typing import Any
 
-from zerver.lib.actions import do_add_realm_filter, do_remove_realm_filter
+from zerver.lib.actions import do_add_linkifier, do_remove_linkifier
 from zerver.lib.management import CommandError, ZulipBaseCommand
-from zerver.models import all_realm_filters
+from zerver.models import linkifiers_from_all_realms
 
 
 class Command(ZulipBaseCommand):
-    help = """Create a link filter rule for the specified realm.
+    help = """Create a linkifier rule for the specified realm.
 
 NOTE: Regexes must be simple enough that they can be easily translated to JavaScript
       RegExp syntax. In addition to JS-compatible syntax, the following features are available:
 
       * Named groups will be converted to numbered groups automatically
       * Inline-regex flags will be stripped, and where possible translated to RegExp-wide flags
 
-Example: ./manage.py realm_filters --realm=zulip --op=add '#(?P<id>[0-9]{2,8})' \
+Example: ./manage.py realm_linkifier --realm=zulip --op=add '#(?P<id>[0-9]{2,8})' \
     'https://support.example.com/ticket/%(id)s'
-Example: ./manage.py realm_filters --realm=zulip --op=remove '#(?P<id>[0-9]{2,8})'
-Example: ./manage.py realm_filters --realm=zulip --op=show
+Example: ./manage.py realm_linkifier --realm=zulip --op=remove '#(?P<id>[0-9]{2,8})'
+Example: ./manage.py realm_linkifier --realm=zulip --op=show
```

Comment:
Let's rename this to `manage.py edit_linkifiers`.

Reply:
Done, renamed to `management/commands/edit_linkifiers.py`